src/api.js

```diff
@@ -8,6 +8,18 @@ function isEmptyObject(obj) {
   return Object.keys(obj).length === 0 && obj.constructor === Object;
 }
 
+function validRequestHooks(requestHooks) {
```

Comment:
I generally prefer names of functions that check/assert conditions to start with `is` or `are`

```suggestion
function areValidRequestHooks(requestHooks) {
```

Reply:
Done, thanks